### PR TITLE
made smal textual changes on the homepage

### PR DESCRIPTION
**Pull request by TakahiroFujitaSJB**

### CODE REVIEW

File: src/pages/HomePage.tsx [hunk 1]
```diff
@@ -60,10 +60,13 @@ function HomePage() {
...
           <p className="regularText">
-            At Tech Start, our members learn new technologies, gain experience
-            with professional development tools and methodologies, and master
-            the art of collaboration. We're also the top spot on campus to meet
-            fellow tech enthusiasts, aspiring entrepreneurs, and innovators.
+            At Tech Start, our members learn to use new technologies, gain
+            experience with professional development tools and methodologies,
+            and master the art of collaboration. You do not need to know the
+            skills before hand! Here at Tech Start, we teach all necessary
+            skills required for you to succeed here and in industry. We're also
```
Comment (Jhappy77) [inline src/pages/HomePage.tsx:67]: Maybe abbreviate this to "We give our members the support and mentorship they need to succeed regardless of their previous experience"

Comment (TakahiroFujitaSJB) [reply src/pages/HomePage.tsx]: Thank you for the feedback! Made the changes just now.

File: src/pages/HomePage.tsx [hunk 2]
```diff
@@ -60,9 +60,11 @@ function HomePage() {
             software, design, and strategy to complete awesome projects.
           </p>
           <p className="regularText">
-            At Tech Start, our members learn new technologies, gain experience
-            with professional development tools and methodologies, and master
-            the art of collaboration. We're also the top spot on campus to meet
+            At Tech Start, our members learn to use new technologies, gain
+            experience with professional development tools and methodologies,
+            and master the art of collaboration. We give our members the support
+            and mentor ship to succeed here and in industry regardless of their
```
Comment (Niyousha99) [inline src/pages/HomePage.tsx:66]: mentorship is one word ;)

**Review by Niyousha99 — APPROVED** [review]
LGTM

**Review by Sahitiakella — APPROVED** [review]
LGTM

**Pull request merged**